uftasks/uftasks-showcase/uftasks-webapp/src/main/java/org/uberfire/client/screens/popup/NewFolderView.java

```diff
@@ -68,16 +79,22 @@ public void show() {
     @Override
     public void hide() {
         modal.hide();
-        folderNameTextBox.setText( "" );
+        folderNameTextBox.setValue( "" );
     }
 
-    @EventHandler("ok-button")
-    public void addFolder( ClickEvent event ) {
-        presenter.newFolder( folderNameTextBox.getText() );
+    @SinkNative( Event.ONCLICK )
+    @EventHandler( "ok-button" )
+    public void addFolder( Event event ) {
+        presenter.newFolder( folderNameTextBox.getValue() );
     }
 
-    @EventHandler("cancel-button")
-    public void cancel( ClickEvent event ) {
+    @EventHandler( "cancel-button" )
+    public void cancel( Event event ) {
         presenter.close();
     }
+
+    @Override
+    public HTMLElement getElement() {
+        return view;
```

Comment:
I don't think this should implement `IsElement` because the `view` element you return here is added as a child to the modal; `view` isn't really the root element. Implementing `IsElement` suggests that consumers of this view can add it as a child to other elements, but that would break the modal usage.